packages/manager/modules/models/user/User.class.js

```diff
@@ -31,4 +31,8 @@ export default class User {
   canHaveInvoicesByPostalMail() {
     return this.billingCountry === 'FR' && this.legalform === 'individual';
   }
+
+  isNotCorporation() {
```

Comment:
Why not use the `isCorporation` and just reverse the check `!user.isCorporation()` ? Imo it seems more logical to check something `is` that `is not`

Reply:
Done